Makefile

```diff
@@ -1,9 +1,33 @@
 # Makefile to build Pokemon Diamond image
 
+.PHONY: clean tidy all default
+
+# Try to include devkitarm if installed
+TOOLCHAIN := $(DEVKITARM)
+
+ifneq (,$(wildcard $(TOOLCHAIN)/base_tools))
+include $(TOOLCHAIN)/base_tools
+endif
+
 ### Default target ###
 
 default: all
 
+# If you are using WSL, it is recommended you build with NOWINE=1.
+NOWINE ?= 0
+
+ifeq ($(OS),Windows_NT)
+EXE := .exe
+WINE := 
+else
+EXE := 
+WINE := "wine "
```

Comment:
Why the quotes? This does not work on my version of ubuntu bionic. It builds OK without the quotes.